lib/tire/index.rb

```diff
@@ -335,7 +335,11 @@ def remove(*args)
       end
       raise ArgumentError, "Please pass a document ID" unless id
 
-      url    = "#{self.url}/#{type}/#{Utils.escape(id)}"
+      options ||= {}
+      options.select! { |k, | [:routing, :parent, :replication, :consistency, :refresh, :timeout].include? k }
```

Comment:
Do we need to add all of these options? I understand `:routing` but why the rest?

Reply:
Those are the options that the endpoint can accept via the 0.90 documentation.